flow/v1/flow.proto

```diff
@@ -655,4 +668,52 @@ message CrossAccountAccessDetail {
 message CreateMessageToSlackResponse {
   string message = 1;
   bool success = 2;
+}
+
+// Response message for the Flow.GetSPToPurchaseDetails rpc.
+message GetSPToPurchaseDetailsResponse {
+  // Recommendation details
+  SavingsPlanRecommendation recommendation = 1;
+  
+  // List of purchase items
+  repeated SavingsPlanPurchaseItem purchases = 2;
+}
+
+// Savings plan recommendation details
+message SavingsPlanRecommendation {
+  string payer_id = 1;
+  string linked_accounts = 2;
+  string account_scope = 3;
+  string savings_plan_type = 4;
+  int32 term = 5;
+  string region = 6;
+  string instance_family = 7;
+  double recommended_hourly_commitment = 8;
+  double purchased_hourly_commitment = 9;
+  string payment_option = 10;
+  double upfront_cost = 11;
+  double monthly_cost = 12;
+  double total_cost = 13;
+  string status = 14;
+  string date_purchased = 15;
```

Comment:
If this is the SP recommendation detail, why is there date_purchase? Also the purchased_hourly_commitment